hw/mcu/nxp/kinetis/MK64F12/pkg.yml

```diff
@@ -23,6 +23,4 @@ pkg.author: "Apache Mynewt <dev@mynewt.apache.org>"
 pkg.homepage: "http://mynewt.apache.org/"
 
 pkg.deps:
-    - "@apache-mynewt-core/hw/hal"
-    - "@apache-mynewt-core/hw/cmsis-core"
```

Comment:
Unrelated to K8x, move into another commit with rationale.

Reply:
Done.